app/presenters/specialist_document_presenter.rb

```diff
@@ -3,6 +3,8 @@ class SpecialistDocumentPresenter < ContentItemPresenter
   include LinkHelper
 
   def headers_for_contents_list_component
+    return [] unless content_item.headers.level_two_headers?
```

Comment:
I think the test for an empty array being returned here needs to be added back in.

Reply:
Done!